app/etc/di.xml

```diff
@@ -485,7 +445,7 @@
     <type name="Magento\Framework\Url">
         <arguments>
             <argument name="session" xsi:type="object">Magento\Framework\Session\Generic\Proxy</argument>
-            <argument name="scopeType" xsi:type="const">Magento\Store\Model\ScopeInterface::SCOPE_STORE</argument>
+            <argument name="scopeType" xsi:type="const">Magento\Framework\App\ScopeInterface::SCOPE_DEFAULT</argument>
         </arguments>
```

Comment:
should be Magento\Store\Model\ScopeInterface::SCOPE_STORE

Reply:
https://github.com/magento/magento2/blob/fd59d87babcaf3e1dd189817fec3d8774cb4213c/app/code/Magento/Store/etc/di.xml#L253

it's overriden with proper value in Magento\Store module